qa/tasks/deepsea.py

```diff
@@ -734,6 +736,9 @@ def __init__(self, ctx, config):
         # cast stage/state_orch value to str because it might be a number
         self.stage = str(self.config.get("stage", ''))
         self.state_orch = str(self.config.get("state_orch", ''))
+        self.alternative_defaults = config.get('alternative_defaults', '')
```

Comment:
The value of `alternative_defaults` is expected to be a list, right? In that case, would it make more sense for the default value here (second argument to `get()` method) to be `[]` instead of `''`?